File: 02-greeter/greeter.f90
! Simple greeter program in Fortran
! Written by M.V.Harish Kumar on 09-05-2024
program greeter
    implicit none
    character(25) :: name

    print '(a$)', "Please enter your name: "
    read(*, '(a)') name

    print '(aaa)', "Hello, ", trim(name), "! Hope you're fine"
    
end program greeter

File: 02-greeter/greeter.ml
(* Simple greeter program in ocaml *)
(* Written by M.V.Harish Kumar on 28-05-2024 *)
open Printf
let () =
  print_string "Please enter your name: ";
  let x = read_line () in
    printf "Hello, %s!Hope you're fine.\n" x